### PR TITLE
Nova: get insecure values from databag

### PR DESCRIPTION
**Pull request by Itxaka**
Instead of doing searches for the insecure values, get them from
the databag

Provides a migration that would look for the actual values
of the insecure attribute and create/update the databag with
them, so the cookbooks will always find the real value in there
instead of defaulting to false

### CODE REVIEW

File: chef/cookbooks/nova/libraries/availability_zone.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ module NovaAvailabilityZone
   def self.fetch_set_az_command_no_arg(node, cookbook_name)
     keystone_settings = KeystoneHelper.keystone_settings(node, cookbook_name)
 
-    nova_insecure = node[:nova][:ssl][:enabled] && node[:nova][:ssl][:insecure]
+    ssl_insecure = BarclampLibrary::Barclamp::Config.load("openstack", "nova")["insecure"] || false
```
Comment (nicolasbock) [inline chef/cookbooks/nova/libraries/availability_zone.rb]: The insecure value is already in the node object, isn't it? Did you add the databag load here for consistency?

Comment (Itxaka) [inline chef/cookbooks/nova/libraries/availability_zone.rb]: indeed. 
Plus when you pick from the databag is already does the whole `component[:ssl] || keystone_settings[:insecure]` which is nice :)

Comment (nicolasbock) [inline chef/cookbooks/nova/libraries/availability_zone.rb]: True!

File: chef/cookbooks/nova/recipes/config.rb [hunk 2]
```diff
@@ -131,10 +129,13 @@
     end
   end
 else
-  cinder_insecure = false
+  use_multipath = false
   keymgr_fixed_key = ""
 end
 
+cinder_config = Barclamp::Config.load("openstack", "cinder", node[:nova][:cinder_instance])
```
Comment (abel-navarro) [inline chef/cookbooks/nova/recipes/config.rb]: still gives me shivers that we relay on cinder for the config although we agreed was the way to go.

**Pull request merged**